launchable/commands/record/commit.py

```diff
@@ -53,16 +53,30 @@ def commit(ctx, source: str, executable: bool, max_days: int, scrub_pii: bool, i
         _import_git_log(import_git_log_output, ctx.obj.dry_run)
         return
 
+    cwd = os.path.abspath(source)
     try:
-        exec_jar(os.path.abspath(source), max_days, ctx.obj.dry_run)
+        is_shallow = subprocess.check_output(
+            ['git', 'rev-parse', '--is-shallow-repository'],
+            stderr=subprocess.DEVNULL,
+            cwd=cwd,
+            universal_newlines=True).strip()
+        if is_shallow == "true":
+            click.echo(click.style(
+                "Can't collect all commits from {} since it is the shallow repository. "
```

Comment:
nits: we don't collect all commits.  I think `Can't collect commit history from ...` is better. WDYT???

Reply:
[Update error messages](https://github.com/launchableinc/cli/pull/574/commits/67a499f6784af7076450df95ef792ca541f32aa6)